### PR TITLE
Example 2 teaches grid-media concept

### PR DESCRIPTION
**Pull request by wardpenney**
screencapture-localhost-4567-examples-1489787689837

### CODE REVIEW

File: source/examples/index.html.slim [hunk 1]
```diff
@@ -101,42 +101,48 @@ section.container.container--spacing
...
-          | This is one of my favorites! It's great for hanging out with the family on a hot August afternoon!
+        = inline_stylesheet :"examples/article-layout"
+        = code_block :"examples/article-layout"
+    figure
+      figcaption SLIM
+      pre
+        code
+          | .article-layout
+              .article-layout__hero
+                // ...content...
```
Comment (whmii) [inline source/examples/index.html.slim]: slim comments are only 1 dash

File: source/assets/stylesheets/examples/article-layout.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+);
+
+$article-layout-grid--tablet-up: (
+  columns: 5,
+  gutter: 2rem,
+  media: "(min-width: 768px)",
+);
+
+.article-layout {
+  @include grid-container($article-layout-grid);
```
Comment (whmii) [inline source/assets/stylesheets/examples/article-layout.scss]: `grid-container` doesn't accept any arguments

**Comment by whmii** [comment]
Minor comments otherwise looks good. We'll probably end up making further revisions to the language in the  future.

File: source/assets/stylesheets/examples/article-layout.scss (hunk 2 above)
Comment (wardpenney) [inline source/assets/stylesheets/examples/article-layout.scss]: @whmii it looks like it does, but is not used. I'll remove from docs and PR on neat

**Pull request merged**